### PR TITLE
Enhancement/1654 analytics component filenames

### PR DESCRIPTION
**Pull request by jqlee85**
## Summary

Contains changes for #1654 for the Analytics store.

Addresses issue #1654 

## Relevant technical choices

Renames component files to match exports. Moves components into a `components` directory inside analytics module.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/activation/ActivationApp.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/**
+ * Activation App component.
```
Comment (felixarntz) [inline assets/js/components/activation/ActivationApp.js:2]: This file shouldn't be part of this PR I think?

File: assets/js/modules/analytics/components/common/AdSenseLinkCta.js [hunk 2]
```diff
@@ -24,9 +24,9 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import CTA from '../../../components/notifications/cta';
+import CTA from '../../../../components/notifications/cta';
 
-export default function AdSenseLinkCTA() {
+export default function AdSenseLinkCta() {
```
Comment (felixarntz) [inline assets/js/modules/analytics/components/common/AdSenseLinkCta.js:29]: Per our naming conventions (all abbreviations fully capitalized), this should continue to be called `AdSenseLinkCTA`.

File: assets/js/modules/analytics/components/common/AnonymizeIpSwitch.js [hunk 3]
```diff
@@ -26,15 +26,15 @@ import { __ } from '@wordpress/i18n';
...
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import Switch from '../../../../components/switch';
+import Link from '../../../../components/link';
+import { sanitizeHTML } from '../../../../util';
 
 const { useSelect, useDispatch } = Data;
 
-export default function AnonymizeIPSwitch() {
+export default function AnonymizeIpSwitch() {
```
Comment (felixarntz) [inline assets/js/modules/analytics/components/common/AnonymizeIpSwitch.js:37]: See above, this should keep its original component name.

File: assets/js/modules/analytics/components/common/AdSenseLinkCta.js (hunk 2 above)
Comment (jqlee85) [reply assets/js/modules/analytics/components/common/AdSenseLinkCta.js]: Ah ok, the pascal case setting in eslintrc.json wasn't allowing those uppercased acronyms, but I'll swap it out for a regex that should allow it.
